test/functional/fixtures/tty-test.c

```diff
@@ -6,6 +6,9 @@
 #include <stdlib.h>
 #include <uv.h>
 
+// -V:STRUCT_CAST:641
+#define STRUCT_CAST(Type, obj) ((Type *)(obj))
```

Comment:
> How?

I mean if, e.g., the type argument to the parameter is changed for some reason in the future.